topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -69,7 +159,7 @@
 
 .pull-right[
 .image-100[
-  ![variation in estimated genome sizes in base pairs](../../images/genomes-size.png)
+  ![variation in estimated genome sizes in base pairs](../../images/genome_assembly_introduction/genomes-size.png)
```

Comment:
Is it intended that you copied the image instead of moving it? (same for others I guess)

Reply:
Yes, as I was not sure of their use in other GTN, I preferred to make a copy than to move to avoid possible problems.

Reply:
You can check that with a big grep, and if it's really used somewhere else, use symlinks to avoid duplicating the files